src/helpers/attrValidation.js

```diff
@@ -154,6 +154,18 @@ export const regex = text => (value) => {
     ) : undefined;
 };
 
+export const validateEmail = text => (value) => {
+  const re = new RegExp(text);
+  return !re.test(value)
+    ? (
+      <FormattedMessage
+        id="validateForm.email"
+        values={{ regex: <b>{text}</b> }}
+        defaultMessage="Invalid email address."
+      />
+    ) : undefined;
+};
+
```

Comment:
in the project we have as dependency [`validator` ](https://www.npmjs.com/package/validator) you can use `isEmail` for a correct validation of the value